### PR TITLE
developer documentation added, env example updated

### PR DESCRIPTION
**Pull request by ProBeta1**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-DB_PASS=
+BOT_TOKEN=(from discord)
+DB_USER=(your mysql db username)
+DB_PASS=(your mysql db pw)
 DB_HOST=127.0.0.1
 DB_PORT=3306
-SONG_DOWNLOAD_DIR=
+DB_KMQ_SETTINGS_TABLE_NAME=kmq
+DB_KPOP_DATA_TABLE_NAME=kpop_videos
+SONG_DOWNLOAD_DIR=(where you want to store songs)
```
Comment (Brainicism) [inline .env.example:8]: `DB_KMQ_SETTINGS_TABLE_NAME`, `DB_KPOP_DATA_TABLE_NAME`, `SONG_DOWNLOAD_DIR` no longer exist

File: docs/DEVELOPER.md [hunk 2]
```diff
@@ -24,28 +28,44 @@ The scripts referenced below can be used as long as the root .env file is set
...
+---
+
 1. `yarn install`
-    - `libsodium` might require the following packages: `autoconf automake g++ libtool`
+   - `libsodium` might require the following packages: `autoconf automake g++ libtool`
 2. Use `*.template` files to create own copy of configuration files in `src/config`
-    - `.env` contains application specific settings. See `.env.example` to see parameters, and `environment.d.ts` to see which are required. 
-3. Apply database migrations for `kmq`. Using `npx knex migrate:latest --knexfile src/config/knexfile_kmq.js`
+   - `.env` contains application specific settings. See `.env.example` to see parameters, and `environment.d.ts` to see which are required.
+3. Apply database migrations for `kmq`. Using `npx knex migrate:latest --knexfile src/config/knexfile_kmq.js` [embedded in scripts]
```
Comment (Brainicism) [inline docs/DEVELOPER.md:38]: We can just remove this step

File: docs/DEVELOPER.md [hunk 3]
```diff
@@ -24,28 +28,44 @@ The scripts referenced below can be used as long as the root .env file is set
...
 4. `npm run dev` to start the bot. Upon first run, the bot will bootstrap the database, as well as download 5 songs to have a minimally working bot
 5. `ts-node src/scripts/download-new-songs` can be used to download the remaining songs in the database
 
-## Debug Mode
-Having every song downloaded may be infeasible for local development. A debug mode can be activated by modifying `src/config/debug_settings.json`, and running in development mode. `forcedSongId` will force the bot to play a specific song given its ID. If `null`, will ignore this setting. `skipSongPlay` will start a game session with a song even if it is not downloaded on the local machine. 
+## Debug Mode - deprecated
```
Comment (Brainicism) [inline docs/DEVELOPER.md:42]: It isn't exactly deprecated, just not maintained. Keep it as it is

File: docs/DEVELOPER.md [hunk 4]
```diff
@@ -24,28 +28,44 @@ The scripts referenced below can be used as long as the root .env file is set
...
-- `download-new-songs.ts {n}`: If a parameter isn't specified, will download every song not currently downloaded. If specified, will download the top `n` most viewed songs according to the database.  
-- `get-unclean-song-names.ts`: Returns a list of potentially unusual song names (hangul, hiragana, strange punctuation) to inspect. Used to check for songs that need aliasing.  
+
+Located under `src/scripts`
+
+- `announce-restart.ts [n]`: Set up a timer for `n` minutes before the bot is stopped. Each server currently playing a game will be notified at regular intervals until the shutdown
+- `download-new-songs.ts {n}`: If a parameter isn't specified, will download every song not currently downloaded. If specified, will download the top `n` most viewed songs according to the database.
+- `get-unclean-song-names.ts`: Returns a list of potentially unusual song names (hangul, hiragana, strange punctuation) to inspect. Used to check for songs that need aliasing.
+
+## Mac specific installation help
```
Comment (Brainicism) [inline docs/DEVELOPER.md:64]: No reason to have Mac specific instructions, users are expected to know how to install these basic packages  from the prerequisites section

**Review by Brainicism — CHANGES_REQUESTED** [review]

**Review by Brainicism — DISMISSED** [review]

**Comment by ProBeta1** [comment]
please discard this pr

**Pull request closed without merging**